src/basilisp/core.lpy

```diff
@@ -4474,7 +4474,16 @@
   The stream must satisfy the interface of :external:py:class:`io.TextIOBase`\\, but
   does not require any pushback capabilities. The default
   ``basilisp.lang.reader.StreamReader`` can wrap any object implementing ``TextIOBase``
-  and provide pushback capabilities."
+  and provide pushback capabilities.
+
+  ``opts`` may include, among other things, the following keys:
+
+  :keyword ``:init-line``:  optional initial line number for reader metadata;
+      helpful for contextualizing errors for chunks of code read from a
+      larger source document.
+  :keyword ``:init-col``: optional initial column number for reader metadata;
```

Comment:
```suggestion
  :keyword ``:init-column``: optional initial column number for reader metadata;
```

I think for this one we should at least makes sure it matches the code.

Reply:
Updated